sidebars.js

```diff
@@ -205,6 +205,10 @@ module.exports = {
         "api/use-case-validiums",
       ],
     },
+    {
```

Comment:
we only need to refer to the id here present in the metadata, and make sure the path is correct, just like the other entries